### ISSUE TITLE
group matching

### ISSUE THREAD

**Issue by psiriders** [oczkers/psub#3]
Hi,

There is a problem with matching in your script so it doesn't recognise any of the files:

 psub  The.X-Files.S01E01.1080p.BRRip.x264-OTURAN.mkv
{'--config': '~/.config/psub.yml',
 '--debug': False,
 '--destination': '.',
 '--help': False,
 '--provider': 'napisy24',
 '--version': False,
 'FILE': ['The.X-Files.S01E01.1080p.BRRip.x264-OTURAN.mkv']}
Filename: The.X-Files.S01E01.1080p.BRRip.x264-OTURAN.mkv
Parsed:
{'category': 'tvshow', 'season': '01', 'episode': '01', 'group': 'oturan', 'title': 'the.x-files'}

[root@matrix Season 6]# psub  Power.2014.S06E01.PROPER.1080p.WEB.h264-CONVOY.mkv
{'--config': '~/.config/psub.yml',
 '--debug': False,
 '--destination': '.',
 '--help': False,
 '--provider': 'napisy24',
 '--version': False,
 'FILE': ['Power.2014.S06E01.PROPER.1080p.WEB.h264-CONVOY.mkv']}
Filename: Power.2014.S06E01.PROPER.1080p.WEB.h264-CONVOY.mkv
Parsed:
{'category': 'tvshow', 'season': '06', 'episode': '01', 'group': 'convoy', 'title': 'power.2014'}

**Comment by oczkers** [comment]
Hi,

it does parse filename correctly but napisy24 has changed they approach so search function has to be updated. Napiprojekt should be working fine (--provider napiprojekt / -p napiprojekt)

**Comment by psiriders** [comment]
True, but for the time being, your script is the only one that supports napisy24 from CLI.
